src/editors/data/redux/thunkActions/problem.js

```diff
@@ -47,15 +48,17 @@ export const getDataFromOlx = ({ rawOLX, rawSettings, defaultSettings }) => {
   return { settings: parsedSettings };
 };
 
-export const loadProblem = ({ rawOLX, rawSettings, defaultSettings }) => (dispatch) => {
+export const loadProblem = ({ rawOLX, rawSettings, defaultSettings }) => (dispatch, getState) => {
   if (isBlankProblem({ rawOLX })) {
     dispatch(actions.problem.setEnableTypeSelection(camelizeKeys(defaultSettings)));
   } else {
-    dispatch(actions.problem.load(getDataFromOlx({ rawOLX, rawSettings, defaultSettings })));
+    const dataFromOlx = getDataFromOlx({ rawOLX, rawSettings, defaultSettings });
+    preprocessProblemData(dataFromOlx, getState());
```

Comment:
This should be happening when the tinyMCE editor loads, so it is not necessary to preprocess the problem here. See https://github.com/openedx/frontend-lib-content-components/blob/main/src/editors/sharedComponents/TinyMceWidget/hooks.js#L74-L97

Reply:
If this is not longer needed, please remove all changes related to `preprocessProblemData`